Main.cpp

```diff
@@ -36,6 +36,7 @@ int main()
                 break;
             default:
                 cout << "Input is not a recognized command. Please enter 0 to exit, or 1, 2, or 3." << endl;
+                userInput = options.promptUserInput();

```

Comment:
This prompt shouldn't be needed, since there is another one on line 43 that is still in the while loop. So it will always prompt when it hits the end of the loop